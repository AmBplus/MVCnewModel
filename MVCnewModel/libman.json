{
  "version": "1.0",
  "defaultProvider": "cdnjs",
  "libraries": [
    {
      "library": "bootstrap@5.1.3",
      "destination": "wwwroot/lib/bootstrap/",
      "files": [
        "js/bootstrap.min.js",
        "css/bootstrap.rtl.min.css"
      ]
    },
    {
      "provider": "cdnjs",
      "library": "jquery@3.6.0",
      "destination": "wwwroot/lib/jquery/",
      "files": [
        "jquery.min.js"
      ]
    }
  ]
}