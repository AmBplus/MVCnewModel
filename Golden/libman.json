{
  "version": "1.0",
  "defaultProvider": "cdnjs",
  "libraries": [
    {
      "library": "bootstrap@5.1.3",
      "destination": "wwwroot/lib/bootstrap/",
      "files": [
        "js/bootstrap.min.js",
        "css/bootstrap.rtl.min.css"
      ]
    },
    {
      "provider": "cdnjs",
      "library": "jquery@3.6.0",
      "destination": "wwwroot/lib/jquery/",
      "files": [
        "jquery.min.js"
      ]
    }
  ,
{
  "provider": "cdnjs",
  "library": "bootstrap@5.1.3",
  "destination": "wwwroot/lib/bootstrap/",
  "files": [
    "css/bootstrap.min.css"
  ]
}
,
{
  "provider": "cdnjs",
  "library": "bootstrap@5.1.3",
  "destination": "wwwroot/lib/bootstrap/"
}

  ,
{
  "provider": "cdnjs",
  "library": "bootstrap@5.1.3",
  "destination": "wwwroot/lib/bootstrap/",
  "files": [
    "js/bootstrap.min.js",
    "css/bootstrap.min.css"
  ]
}
]
}